deno/payloads/v10/channel.ts

```diff
@@ -145,7 +150,7 @@ export interface APIVoiceChannelBase<T extends ChannelType> extends APIGuildChan
 
 export interface APIGuildVoiceChannel
 	extends APIVoiceChannelBase<ChannelType.GuildVoice>,
-		Omit<APITextBasedChannel<ChannelType.GuildVoice>, 'name'> {
+		Omit<APITextBasedChannel<ChannelType.GuildVoice>, 'name' | 'last_pin_timestamp'> {
```

Comment:
Maybe this shouldn't extend TextBasedChannel instead?

Reply:
I'm kind of unsure here. The fact messages can't be pinned in a text channel is unique to voice channels, so in my head it made sense to exclude it here specifically but keep it for all other channel types - avoiding duplicates on the way.

I can remove it from TextBasedChannel and add it individually to more specific types, if you prefer.